libs/common/src/constants/e2e-fixtures.ts

```diff
@@ -0,0 +1,24 @@
+export const PORTAL_USER_SIGN_IN_TOKENS = {
+  VALID_FORMAT_SIGN_IN_TOKEN_ONE: '1111111111abcdef1111111111abcdef1111111111abcdef1111111111abcdef',
+  VALID_FORMAT_SIGN_IN_TOKEN_TWO: '3333333333abcdef3333333333abcdef3333333333abcdef3333333333abcdef',
+  VALID_FORMAT_SIGN_IN_TOKEN_THREE: '4444444444abcdef4444444444abcdef4444444444abcdef4444444444abcdef',
+  VALID_FORMAT_SIGN_IN_TOKEN_FOUR: '5555555555abcdef5555555555abcdef5555555555abcdef5555555555abcdef',
+  INVALID_FORMAT_SIGN_IN_TOKEN: '2222222222abcdef2222222222abcdef',
+};
+
+export const NODE_TASKS = {
+  GET_ALL_BANKS: 'getAllBanks',
+  INSERT_UTILISATION_REPORTS_INTO_DB: 'insertUtilisationReportsIntoDb',
+  REMOVE_ALL_UTILISATION_REPORTS_FROM_DB: 'removeAllUtilisationReportsFromDb',
+  REMOVE_ALL_PAYMENT_MATCHING_TOLERANCES_FROM_DB: 'removeAllPaymentMatchingTolerancesFromDb',
+  INSERT_PAYMENT_MATCHING_TOLERANCES_INTO_DB: 'insertPaymentMatchingTolerancesIntoDb',
+  REINSERT_ZERO_THRESHOLD_PAYMENT_MATCHING_TOLERANCES: 'reinsertZeroThresholdPaymentMatchingTolerances',
+  INSERT_FEE_RECORDS_INTO_DB: 'insertFeeRecordsIntoDb',
+  INSERT_PAYMENTS_INTO_DB: 'insertPaymentsIntoDb',
+  REMOVE_ALL_PAYMENTS_FROM_DB: 'removeAllPaymentsFromDb',
+  REMOVE_ALL_FEE_RECORDS_FROM_DB: 'removeAllFeeRecordsFromDb',
+  DELETE_ALL_FROM_SQL_DB: 'deleteAllFromSqlDb',
+  INSERT_TFM_FACILITIES_INTO_DB: 'insertManyTfmFacilities',
+  DELETE_ALL_TFM_FACILITIES_FROM_DB: 'deleteAllTfmFacilities',
+  REMOVE_ALL_FEE_RECORD_CORRECTION_TRANSIENT_FORM_DATA_FROM_DB: 'removeAllFeeRecordCorrectionTransientFormDataFromDb',
+};
```

Comment:
It looks like we can remove instances of `PORTAL_USER_SIGN_IN_TOKENS` and `NODE_TASKS` from `e2e-tests/e2e-fixtures` now that this is in libs/common

Reply:
Actually, it looks like there are a few constants we can remove from `e2e-tests/e2e-fixtures` and instead, import the constants in individual tests from `lib/common`

I think we should merge this PR and create a ticket/subtask to do this straight afterwards. Conscious this PR has been open for a while/had delays etc and is blocking

What do you think @rkontogianni ?

Reply:
Discussed - created DTFS2-7695 to tackle this.